tests/suites/test_suite_psa_crypto.function

```diff
@@ -3668,6 +3668,7 @@ void aead_decrypt( int key_type_arg, data_t *key_data,
     if( expected_result != PSA_ERROR_INVALID_ARGUMENT )
         TEST_EQUAL( output_size,
                     PSA_AEAD_DECRYPT_OUTPUT_SIZE( alg, input_data->len ) );
+    TEST_ASSERT( output_size > 0 );
```

Comment:
No. If we're decrypting an empty plaintext, `output_size` should be 0, and `output_data` will be `NULL` as desired.

Reply:
Ok, thanks for clarifying.